content/courses/Ada_For_The_C_Embedded_Developer/chapters/08_Performance.rst

```diff
@@ -141,6 +141,17 @@ inline a subprogram:
 
 - when it contains tasks, etc.
 
+In addition to the :ada:`Inline` aspect, we also have the :ada:`Inline_Always`
```

Comment:
In retrospect maybe we shouldn't even mention Inline_Always. It's a fairly advanced topic.